mysql-test/suite/innodb/include/temp_table_encrypt.inc

```diff
@@ -6,7 +6,7 @@ call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find master key,
 
 # we need restart to make sure keyring settings are picked up
 let $restart_hide_args = 1;
-let $restart_parameters = $KEYRING_RESTART_PARAM --innodb_max_dirty_pages_pct=0;
+let $restart_parameters = $KEYRING_RESTART_PARAM --innodb_max_dirty_pages_pct=0 --innodb_max_dirty_pages_pct_lwm=0;
```

Comment:
Why need to disable preflushing?

Reply:
There is warning because of max_dirty_pages_pct = 0 lwm cannot be set to 10 (default) and it auto adjusts to 0.